src/posts/paper/14-research-sources.ts

```diff
@@ -163,7 +163,9 @@ Another important thing to know about the infectious period is that only about 9
 
 ![He et al](${heExtended})
 
-(If you were previously familiar with this source, note that it was substantially updated in a [07 Aug 2020 author correction](https://www.nature.com/articles/s41591-020-1016-z).)
+*(If you were previously familiar with this source, note that it was substantially updated in a [07 Aug 2020 author correction](https://www.nature.com/articles/s41591-020-1016-z).)*
+
+One insight we can gain is that since 50% of transmission occurrs after symptoms onset, if you confirm with the person you are seeing that they don't have symptoms right before you interact, you are reducing your risk by 50%.
```

Comment:
Is research sources a place people go for these things? Is this linked from anywhere? Otherwise I'd imagine FAQ would make more sense. Or possibly there should be a page for spreadsheet feature sources.

Reply:
Yeah it's linked from a bunch of places, I think.

I could see this being in the FAQ as well. It just seems to fit here since all the surrounding context is here.

What would the FAQ question be?

Reply:
Let me know if you'd like a change here.